ykrt/src/mt.rs

```diff
@@ -242,8 +306,8 @@ impl MTThread {
                             // count further.
                             return None;
                         }
-                        let new_pack = self.inner.tid as usize | PHASE_TRACING;
-                        if loc.pack.compare_and_swap(lp, new_pack, Ordering::Release) == lp {
+                        let new_state = self.inner.tid as usize | PHASE_TRACING;
```

Comment:
Did we say we always use `usize::from` from now on, or this is case an exception?

Reply:
This is a pointer -> usize conversion, so I think we have to use `as` unfortunately.

Reply:
Ah, that's fine then.